packages/stark-ui/src/modules/minimap/components/minimap.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="stark-minimap">
```

Comment:
Now it is not necessary to add the "stark-xxxx" class anymore... we should do this via Angular by adding this to the component class:

```typescript
@Component({
 	selector: "stark-minimap",
 	templateUrl: "./minimap.component.html",
 	encapsulation: ViewEncapsulation.None,
        host: {
             class: componentName
        }
 })
 export class StarkMinimapComponent implements OnInit, OnChanges, OnDestroy {
      ...
```

You can even remove this div at the end.

By doing all this, you might need to adapt the CSS rules as well since the hierarchy of HTML elements changed